weather_sp/splitter_pipeline/file_splitters.py

```diff
@@ -90,89 +85,132 @@ def should_skip(self):
 
 class GribSplitter(FileSplitter):
 
+    def __init__(self, input_path: str, output_info: OutFileInfo,
+                 force_split: bool = False, logging_level: int = logging.INFO):
+        super().__init__(input_path, output_info,
+                         force_split, logging_level)
+        self.output_info.set_formatting_if_needed('_{typeOfLevel}_{shortName}')
```

Comment:
I'm not sure I understand how you want this here. My goal was to set this in a way that does not accidentally overwrite the value the user has specified, and also to avoid repeatedly having to check whether the default is needed.

Reply:
Removed this. No longer needed if we don't set defaults.